source/common/http/headers.h

```diff
@@ -212,6 +215,14 @@ class HeaderValues {
     const std::string AcceptEncoding{"Accept-Encoding"};
     const std::string Wildcard{"*"};
   } VaryValues;
+
+  struct {
+    const std::string AllowHeadersHystrix{"Accept, Cache-Control, X-Requested-With, Last-Event-ID"};
```

Comment:
nit: I'm not sure I would define this in this file as it is specific to an extension. Can you move this to somewhere in your extension code?